frontend/lib/resolution/resolution-error-classes-list.cpp

```diff
@@ -328,6 +328,14 @@ void ErrorDotExprInUseImport::write(ErrorWriterBase& wr) const {
       "'use' or 'import'.");
 }
 
+void ErrorDotTypeOnType::write(ErrorWriterBase& wr) const {
+  auto dot = std::get<const uast::Dot*>(info);
+  auto dottedType = dot->receiver();
+  wr.heading(kind_, type_, dot, "can't apply '.type' to a type (",
+             dottedType, ").");
+  wr.code(dottedType, {dot});
```

Comment:
Have you printed this in detailed mode? I think this would underline `thewhole.expression`, but print only the line with `thewhole`. The `code` command takes a location to _print_ (should probably be `dot`) and a list of locations to _underline_ (in this case, probably the `dottedType`?). Or maybe you want to print and underline the whole expression, in which case it's `wr.code(dot, { dot })`.

Reply:
I tried it in detailed mode ([for the test I added](https://github.com/chapel-lang/chapel/pull/24068/files#diff-f2d2dc7a3e9cee4d985d73ac247de467a41bf31b2958161698e0607197a63753)) and it printed and underlined the whole thing, but that's probably because they happen to be on the same line. I think I want the whole expression printed and underlined, so I'll change to `wr.code(dot, { dot })`.